internal/kafka/internal/services/data_plane_cluster_test.go

```diff
@@ -960,6 +945,10 @@ func Test_DataPlaneCluster_setClusterStatus(t *testing.T) {
 
 	for _, tt := range cases {
 		t.Run(tt.name, func(t *testing.T) {
+			if tt.skip {
+				t.Skip("skipping dynamic scaling related tests")
+			}
+
 			f, spyReceivedStatus := tt.inputFactory()
 			if f == nil {
 				t.Fatalf("dataPlaneClusterService is nil")
```

Comment:
Same as previous comment. This should be:
```suggestion
			Expect(f).ToNot(BeNil(), "dataPlaneClusterService is nil")
```